x-pack/plugins/console_extensions/spec/overrides/data_frame.put_data_frame_transform.json

```diff
@@ -0,0 +1,25 @@
+{
+  "data_frame.put_data_frame_transform": {
+    "data_autocomplete_rules": {
+      "source": "SOURCE_INDEX_NAME",
+      "dest": "DEST_INDEX_NAME",
+      "pivot": {
+        "group_by": {
+          "__template": {
+            "NAME": {
+              "terms": {
```

Comment:
this can be "terms", "histogram" or "date_histogram"

Reply:
yup, fixed that